src/components/workspaces/boards-list.style.ts

```diff
@@ -6,15 +6,18 @@ export const WsSection = styled.section`
   grid-template-columns: 1fr 8fr;
   padding: 3em 0;
 `;
-export const WsNav = styled.nav`
-  h3 {
-    font-size: 1.3em;
-  }
-  ul {
-    list-style-type: none;
-  }
+export const WsNav = styled.nav``;
```

Comment:
If you don't add any style to the tag, I think you don't have to create a styled-component. You can just use pure HTML tag instead 👍